driver-sync/src/test/functional/com/mongodb/internal/connection/OidcAuthenticationProseTests.java

```diff
@@ -268,14 +272,19 @@ private static Stream<Arguments> testValidCallbackInputsTimeoutWhenTimeoutMsIsSe
     }
 
     // Not a prose test
-    @ParameterizedTest(name = "test callback timeout when server selection timeout is "
-            + "infinite and timeoutMs is set to {0}")
-    @ValueSource(ints = {0, 100})
-    void testCallbackTimeoutWhenServerSelectionTimeoutIsInfiniteTimeoutMsIsSet(final int timeoutMs) {
+    @Test
+    @DisplayName("test callback timeout when server selection timeout and "
+            + "timeoutMs are infinite")
+    void testCallbackTimeoutWhenServerSelectionTimeoutIsInfiniteTimeoutMsIsSet() {
```

Comment:
`timeoutMs`/`server selection timeout` represent neither the names of the `ConnectionString` options, nor names of the API methods, which makes the naming both inconsistent with the existing test code, and undiscoverable.

```suggestion
    @Test
    @DisplayName("test callback timeout when serverSelectionTimeoutMS and timeoutMS are infinite")
    void testCallbackTimeoutWhenServerSelectionTimeoutMsIsInfiniteTimeoutMsIsSet() {
```